test/index.ts

```diff
@@ -389,4 +389,22 @@ describe('teeny', () => {
       }
     );
   });
+
+  it('should respect followRedirect option being disabled', done => {
+    const scope = nock(uri).get('/').reply(302, '', {
```

Comment:
this test is accurate, but for full `request` compatibility, by default the following methods do NOT follow redirects unless `followAllRedirects` is enabled: `PATCH`, `PUT`, `POST`, and `DELETE`.
see https://github.com/request/request/blob/536f0e76b249e4545c3ba2ac75e643146ebf3824/lib/redirect.js#L56-L62